src/test-utils/loadingHelpers.ts

```diff
@@ -0,0 +1,66 @@
+import { screen, waitFor } from '@testing-library/react'
+
+export const waitForPageLoad = async (
+  options: {
+    timeout?: number
+    waitForElement?: () => Promise<HTMLElement>
+    loadingIndicator?: string
+  } = {},
+) => {
+  const { timeout = 15000, waitForElement, loadingIndicator = 'Loading component...' } = options
```

Comment:
This seems to duplicate functionality that already exists in testing-library, as well as introducing a longer timeout that sets a higher global ceiling on test run times. What problem is this solving?